web/pdf_find_controller.js

```diff
@@ -63,7 +63,6 @@ class PDFFindController {
 
   reset() {
     this.extractTextPromises = [];
-    this.pendingFindMatches = Object.create(null);
```

Comment:
Done in the new commit series. Thank you for finding this, since I did not encounter it in my local testing unfortunately!

Reply:
No worries; I'm guessing that you've already figured out why that commit didn't work, but otherwise please let me know and I'll try my best to explain what the problem was.